my-app/src/css/bootstrap-theme.css

```diff
@@ -0,0 +1,587 @@
+/*!
```

Comment:
Can you link to a url that hosts this file, rather than include it in your repo. Do this for all the large files.